docs/site/Controller-generator.md

```diff
@@ -100,25 +104,25 @@ export class TodoController {
     return await this.todoRepository.create(obj);
   }
 
-  @get('/todo/count')
+  @get('/todos/count')
   async count(@param.query.string('where') where: Where): Promise<number> {
     return await this.todoRepository.count(where);
   }
 
-  @get('/todo')
+  @get('/todos')
   async find(@param.query.string('filter') filter: Filter): Promise<Todo[]> {
     return await this.todoRepository.find(filter);
   }
 
-  @patch('/todo')
+  @patch('/todos')
   async updateAll(
     @param.query.string('where') where: Where,
     @reqeustBody() obj: Todo,
```

Comment:
Todo[]

Reply:
https://github.com/strongloop/loopback-next/blob/master/packages/repository/src/repositories/legacy-juggler-bridge.ts#L169
Seems like it only takes in a single instance of `Todo`